src/__tests__/queries.find.js

```diff
@@ -56,6 +57,13 @@ test('find asynchronously finds elements', async () => {
   await expect(findByRole('dialog')).resolves.toBeTruthy()
   await expect(findAllByRole('dialog')).resolves.toHaveLength(1)
 
+  await expect(
+    findByRole('progressbar', {queryFallbacks: true}),
+  ).resolves.toBeTruthy()
+  await expect(
+    findAllByRole('progressbar', {queryFallbacks: true}),
+  ).resolves.toHaveLength(1)
```

Comment:
I can't make a suggestion in GitHub for this, but I would suggest this change to:

```javascript
  await expect(findByRole('meter')).resolves.toBeTruthy()
  await expect(findAllByRole('meter')).resolves.toHaveLength(1)
  await expect(findByRole('progressbar', {queryFallbacks: true})).resolves.toBeTruthy()
  await expect(findAllByRole('progressbar', {queryFallbacks: true})).resolves.toHaveLength(1)
```

Thanks!